validators/cgroup_validator.go

```diff
@@ -81,7 +108,25 @@ func (c *CgroupsValidator) validateCgroupSubsystems(cgroups, subsystems []string
 
 }
 
-func (c *CgroupsValidator) getCgroupSubsystems() ([]string, error) {
+func (c *CgroupsValidator) getCgroupv2Subsystems() ([]string, error) {
```

Comment:
getCgroupv2Subsystems -> getCgroupV2Subsystems

Reply:
please move the v2 function bellow the v1 function.

Reply:
Fixed it